### PR TITLE
EC-692 Show Submitted Date on Review and Submitted pages

### PR DESCRIPTION
**Pull request by AndyChildNice**

### CODE REVIEW

File: Comments/ClientApp/src/components/Review/Review.js [hunk 1]
```diff
@@ -45,7 +46,7 @@ type PropsType = {
 type StateType = {
 	consultationData: ConsultationDataType | null,
 	commentsData: ReviewPageViewModelType | null,
-	userHasSubmitted: boolean,
+	submittedDate: boolean,
```
Comment (josealmeida) [inline Comments/ClientApp/src/components/Review/Review.js]: Should this be of type Date?

File: Comments/ClientApp/src/components/Review/Review.js [hunk 2]
```diff
@@ -72,7 +73,7 @@ export class Review extends Component<PropsType, StateType> {
 			loading: true,
 			consultationData: null,
 			commentsData: null,
-			userHasSubmitted: false,
+			submittedDate: false,
```
Comment (josealmeida) [inline Comments/ClientApp/src/components/Review/Review.js]: Should this be null?

File: Comments/ClientApp/flow-typed/type-defs.js [hunk 3]
```diff
@@ -47,6 +47,7 @@ declare type ConsultationStateType = {
 	supportsDownload: boolean,
 	reference?: string,
 	userHasSubmitted: boolean,
```
Comment (josealmeida) [inline Comments/ClientApp/flow-typed/type-defs.js]: We might want to remove userHasSubmitted as it's become deprecated

File: Comments/ClientApp/src/components/Review/Review.js (hunk 1 above)
Comment (AndyChildNice) [reply Comments/ClientApp/src/components/Review/Review.js]: Has been changed to Date as per suggestion.

File: Comments/ClientApp/src/components/Review/Review.js (hunk 2 above)
Comment (AndyChildNice) [reply Comments/ClientApp/src/components/Review/Review.js]: Has been changed to null as per suggestion.

File: Comments/ClientApp/flow-typed/type-defs.js (hunk 3 above)
Comment (AndyChildNice) [reply Comments/ClientApp/flow-typed/type-defs.js]: Has been removed as per suggestion.

**Pull request merged**